### ISSUE TITLE
Failed to transform request with plugins: req_304fdb774c664e1992b877449d88076c

### ISSUE THREAD

**Issue by dscham** [dp-152/insomnia-plugins#8]
Hi there,

I love the idea of this plugin, especially because I have to send a lot of multiline strings. Which I didn't find an efficient way to convert for JSON yet. And writing my own tool is also kind of overkill, for my task at hand.

There is nothing secret in this, so I'm posting the YAML I entered here. Resulting in the error from the title, which is very useless and I couldn't find more detailed info. Maybe you have an idea why I get it.

```yaml
#instanceid 202172525
iamgeId: "4efbc0ba-2313-4fe1-842a-516f8652e729"

userData: |
  #cloud-config
  #hostname
  fqdn: dscntbvm2.dscntblan1

  #packages
  package_update: true
  package_upgrade: true
  package_reboot_if_required: true

  packages:
    - zsh
    - git

  #config
  disable_root: true
  locale: en_US
  timezone: Europe/Berlin

  #users
  user: 
    - name: root
      shell: /bin/zsh
  users:
    - default
    - name: dsc
      ssh_authorized_keys:
        - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC7GiAl8QFjKgRvcxe/stjrjW8cv3+14w5rnNPaAs7CLATadt1kEk0Kvo0GmNjTywxWznZJs0l81BL3YqkbVnaYe5sAXEljTM+ORYFDlFlcLb4gdBSTtIaMfo+CZ5TVkDoYPJwGqj2a52vSZRoowx7iWb14C5IyFlF4GmI4mRgyA9HqSr17vpf6mEUpUuO6cwQZI0vBRWcvLaXfesFITK1bZ8xDxSt+Juq39kjocMPIWO43Nk/fAbVJd0tg5hwSROxbEOrOE6/6YDA0l5ZomKoytb9+wJNU5Ba0c38u2LOK7CItv/LAH7V1Hqsb3Nq6BHl9v/o6q8N+pZXKYg0DkNScn1uzQbs/SQUwXzTwXaHG/Ai5bGxdjEMGPSj4cxoZIJbE7KGazUGRl7lTXwhCEkeuwRLxSX76axpsI5l561MQeERDOJiswhAZqfQEKQJMAYjeFMjDDYs+HcKxYXSEWxLkIPn21P9onhI9CQvGdwLefXNBsXK61BYbdXCoObHf3es= dscham@dsnb1
      sudo: ["ALL=(ALL) NOPASSWD:ALL"]
      lock_passwd: true
      shell: /bin/zsh
```

**Comment by mz-alm** [comment]
Thanks a lot for reporting the issue! I'm sorry I couldn't get back to you sooner, work has been keeping me quite busy these last few weeks.

I ran a few tests here and it doesn't seem like I can reproduce the issue with the payload you provided. This was tested against a simple server that just echoes back the POST'ed payload as a JSON:
![image](https://github.com/user-attachments/assets/78c9ccd4-8943-4cec-9767-a20b813a7e06)

Would you be able to confirm the plugin version you have installed? [The latest version should be 0.1.2](https://github.com/dp-152/insomnia-plugins/releases/tag/insomnia-plugin-yaml-as-json%400.1.2), released on Aug. 26.

In case you're already on the latest version, please check on the dev tools console (View -> Toggle DevTools -> Console) for any messages prefixed with `[yaml-as-json]` for any clues and report back, if possible. Those are the log messages emitted by the plugin, and they may have a clue as to what might be happening to you.
![image](https://github.com/user-attachments/assets/b3b68f98-7d9a-40ce-a6c9-cc00410feec9)

**Comment by dscham** [comment]
Interesting, I installed via the Insomnia Plugin hub, which has 0.1.2 But it's from 26th August, according to the plugin page.

Edit: Dumb me, that's the release date. Updated says 26th.

Edit2: I have no other plugins and Insomnia is version 2023.3.0. Oh wait, that's really old. Haven't used it in a while on my private PC. Welp. That's it then, probably.

Edit3: I already did what I wanted manually in the end. It was just 5 servers, so that was okay. Therefore, I'll close this issue.
Thanks for looking into it.
